### PR TITLE
Add a message to the comments box when Approve or a Reject action is selected.

### LINKED ISSUE #19981: Prepopulate empty review message in reviewer tools for certain actions
On a reviewer tools review page, if the review text is empty (!) and

* The reviewer selects "Approve" as an action, prepopulate the text area with

"Thank you for your contribution."

* The reviewer selects "Reject" or "Reject multiple versions", prepopulate the text area with

"This add-on didn't pass review because of the following problems:

1)"

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #19981 

This is a quick and dirty way of implementing this. If we don't need these messages to be translated this might be good enough. Otherwise we probably have to get tricky with the Django form.

### CODE REVIEW

File: static/js/zamboni/reviewers.js [hunk 1]
```diff
@@ -60,13 +60,27 @@ function initReviewActions() {
...
+    // Input message into empty comments textbox.
+    if ($comments.val().length === 0) {
+      if ($element.text().trim() === 'Approve') {
+        insertAtCursor($comments, 'Thank you for your contribution.');
+      } else if (
+        ['Reject', 'Reject Multiple Versions'].includes($element.text().trim())
+      ) {
+        insertAtCursor(
+          $comments,
+          "This add-on didn't pass review because of the following problems:\n\n1)",
```
Comment (wagnerand) [inline static/js/zamboni/reviewers.js:80]:
```suggestion
          "This add-on didn't pass review because of the following problems:\n\n1) ",
```

Comment (wagnerand) [reply static/js/zamboni/reviewers.js:80]: Nit, which I missed in the issue. Thanks!

**Review by diox — CHANGES_REQUESTED** [review]
It's okay to hardcode what text to insert in JavaScript, but I wouldn't rely on element text for this, as it could change or be translated, breaking the feature.

I would pass `data-` attributes instead as we've done for other reviewers features. In fact you could have the whole string you're adding set on the relevant action in `ReviewHelper` and then pass that as a `data-` attribute.

**Comment by bobsilverberg** [comment]
> It's okay to hardcode what text to insert in JavaScript, but I wouldn't rely on element text for this, as it could change or be translated, breaking the feature.
> 
> I would pass `data-` attributes instead as we've done for other reviewers features. In fact you could have the whole string you're adding set on the relevant action in `ReviewHelper` and then pass that as a `data-` attribute.

I have changed the code to store the boilerplate text in `ReviewHelper` and then pass it through as `data-value`.

File: src/olympia/reviewers/utils.py [hunk 2]
```diff
@@ -612,6 +612,11 @@ def get_actions(self):
             can_approve_multiple = False
 
         # Definitions for all actions.
+        boilerplate_for_approve = _('Thank you for your contribution.')
+        boilerplate_for_reject = _(
+            "This add-on didn't pass review because of the following problems:\n\n1) "
+        )
```
Comment (diox) [inline src/olympia/reviewers/utils.py:618]: @wagnerand do we want to translate those ? Even if we did, that would be in the reviewer's language and not the developer, so I'm thinking no.

**Review by diox — APPROVED** [review]
:+1: but double-check with Andreas about translating the text, I don't think we want that.

File: src/olympia/reviewers/utils.py (hunk 2 above)
Comment (bobsilverberg) [reply src/olympia/reviewers/utils.py:618]: That makes sense to me too. I will remove the translations of those strings, but we can also see if Andreas disagrees with that.

Comment (wagnerand) [reply src/olympia/reviewers/utils.py]: We don't want to translate these. All reviewer communication is in English.

**Pull request merged**